pkg/defaults/amd64.go

```diff
@@ -44,3 +44,15 @@ func setDefaultAmd64DisksBus(spec *v1.VirtualMachineInstanceSpec) {
 func SetAmd64Defaults(spec *v1.VirtualMachineInstanceSpec) {
 	setDefaultAmd64DisksBus(spec)
 }
+
+func SetAmd64Defaults_Watchdog(obj *v1.Watchdog) {
```

Comment:
This is not called anywhere. You should call it from `SetAmd64Defaults`.
Additionally you can shorten the name to `SetAmd64Watchdog`

Reply:
Addressed.